packages/biome-config/biome.json

```
@@ -110,41 +132,61 @@
 			"module"
 		]
 	},
+	"css": {
+		"formatter": {
+			"enabled": true,
+			"quoteStyle": "single"
+		},
+		"linter": {
+			"enabled": true
+		},
+		"parser": {
+			"cssModules": true
+		}
+	},
+	"json": {
+		"linter": {
+			"enabled": true
+		}
+	},
+	"assist": {
+		"enabled": true,
+		"actions": {
+			"source": {
+				"organizeImports": "on",
+				"useSortedKeys": "on",
+				"useSortedAttributes": "on"
+			}
+		}
+	},
 	"overrides": [
 		{
-			"include": ["**/*.{js,jsx,mjs,cjs,ts,tsx,cts,mts}"],
+			"includes": ["**/*.{js,jsx,mjs,cjs,ts,tsx,cts,mts}"],
```

We should already exclude `node_modules` here. Since they are commonly used.

```suggestion
			"include": ["**/*.{js,jsx,mjs,cjs,ts,tsx,cts,mts}", "!**/node_modules"],
```